src/lib/blueprint.ts

```diff
@@ -851,18 +916,100 @@ const createParameter = (
       return { ...baseParam, format: 'string', jsonType: 'string' }
     case 'boolean':
       return { ...baseParam, format: 'boolean', jsonType: 'boolean' }
-    case 'array':
-      return { ...baseParam, format: 'list', jsonType: 'array' }
+    case 'array': {
```

Comment:
Can we move this case handler to a function? `createArrayParameter`? This case is really long now and even has a nested function and a nested switch statement.

Reply:
4d33c5a6ec8534f2418c67bd5da20f77c595a584